Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -380,9 +387,44 @@ def checkFileLength (lines : Array String) (existingLimit : Option ℕ) : Option
 
 end
 
+
+section unicodeLinter
+
+/- Hashable instance for use with the unicodeLinter -/
+local instance : Hashable Char where
+  hash c := c.val.toUInt64
+
+/-- TODO make complete and order nicely -/
```

Comment:
Yes, ordering this would be great. I presume this contains numbers, all ASCII letters, space etc., but it's very hard for me to check (short of using the search function ~60 times).